src/components/Header/Header.tsx

```diff
@@ -28,10 +28,10 @@ export const Header = () => {
     },
     { name: t(Translation.NAVBAR_PROFILE), href: '/profile/settings', current: false },
   ]
-
   const isAuthenticated = useAppSelector(usersSelector.isAuthenticated)
   const isLogOutLoading = useAppSelector(usersSelector.isLogOutLoading)
-
+  const { role } = useAppSelector(usersSelector.user)
+  const isAdmin: boolean = role !== Roles.Admin && role !== Roles.NOROLE
```

Comment:
role === Roles.Admin?

Reply:
I'm not entirely certain if I understood you correctly. What appears to be the issue here?